Upstream: no, seems pretty dead
Reason: undefined references to pow and log10 otherwise

From dbeca7d4af67507c5c3903ceffad679d0bb77fdb Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Sat, 16 Jan 2021 17:04:14 +0100
Subject: [PATCH] libmpcdec: Link against libm

...for pow and log10.
---
 libmpcdec/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libmpcdec/CMakeLists.txt b/libmpcdec/CMakeLists.txt
index 0bc5b62..885fa18 100755
--- a/libmpcdec/CMakeLists.txt
+++ b/libmpcdec/CMakeLists.txt
@@ -5,3 +5,5 @@ else(SHARED)
   add_library(mpcdec_static STATIC huffman mpc_decoder mpc_reader streaminfo mpc_bits_reader mpc_demux requant synth_filter ${libmpc_SOURCE_DIR}/common/crc32)
 endif(SHARED)
 
+target_link_libraries(mpcdec m)
+
-- 
2.30.0

